### DISCUSSION TITLE
Avoid OnInsert OnModify OnDelete - TableExtensions

### DISCUSSION THREAD

**Discussion by pri-kise** [microsoft/alguidelines#205] [Q&A]
Could someone clarify if there is a difference between an OnInsert Trigger on TableExtensions vs. an OnInsert EventSubscriber concerning performance considerations.

https://github.com/microsoft/alguidelines/blob/main/content/docs/BestPractices/SubscriberCodeunits/index.md#avoid-oninsertonmodifyondelete

**Comment by waldo1001** [comment]
This is indeed something that we should look into .. I got that as feedback from my session/workshops  @navtechdays as well.

Question is - what's there to test.   And how to test 🤔.  Something like:
- does OnInsert on Tableext also break BulkInsert when Insert(false)
- does OnModify on TableExt also break ModifyAll when MOdify(false)
- ... (what else?)
